versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.14"
+
+  required_providers {
+    helm = {
+      source  = "hashicorp/helm"
+      version = "~> 2.1"
```

```suggestion
      version = ">= 2.1"
```

Use less restrictive version constraints in modules.

---

fixed